pytest_cloud/rsync.py

```diff
@@ -86,7 +86,7 @@ def send(self, raises=True):
                     '--inplace '
                     '--delete-excluded '
                     '--delete '
-                    '-e \"ssh -T -c arcfour -o Compression=no -x\" '
```

Comment:
as I remember, this cipher was passed to have the least possible network overhead, so what is the next another cipher we can pass from the supported ones?

Reply:
OK, that makes sense. Let me look around to see which cipher should replace `arcfour`.

Reply:
Based on this old post: https://blog.famzah.net/2010/06/11/openssh-ciphers-performance-benchmark/ indeed the best was `arcfour`. But later, author updated his benchmark https://blog.famzah.net/2015/06/26/openssh-ciphers-performance-benchmark-update-2015/ and found that the fastest is `aes128-gcm@openssh.com`.

According to analysis, everything depends on type of platform. AES instruction set, introduced by Intel in 2010 and AMD in mid-2013 is significantly improving speeds of operations.

Do you think we should force ssh  to use this cipher?

Reply:
let's use aes128-gcm@openssh.com then
the point of pytest-cloud that it's an opinionated approach, so the less to configure for the user is better

Reply:
Replaced with `aes128-gcm@openssh.com`. Thanks

Reply:
can't this option be just confuguration with sane default?

Reply:
makes sense

Reply:
OK, I'll try making it config option.

Reply:
@bubenkoff @spinus is it looking better now?